apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveRoutes.kt

```diff
@@ -33,6 +37,22 @@ internal fun Route.oppgaveRoutes(service: OppgaveService) {
                 else -> call.respond(HttpStatusCode.Forbidden)
             }
         }
+
+        get("/gosys") {
+            when (brukerTokenInfo) {
+                is Saksbehandler -> {
+                    val tema = call.request.queryParameters["tema"] as String
+                    val enhetsnr = call.request.queryParameters["enhetsnr"] as String
+
+                    val oppgaver = runCatching {
+                        gosysOppgaveKlient.hentOppgaver(tema, enhetsnr, brukerTokenInfo)
+                    }.onFailure { logger.error(it.message, it) }
+
+                    call.respond(HttpStatusCode.OK, oppgaver)
```

Comment:
Hvordan fungerer denne responsen -- `oppgaver` er vel en `Result<GosysOppgaver>` her, men løfter `ktor` det ut? Og hvis den er failure, burde statuskoden være `500`.

Reply:
_oppgaver_ er en `List<GosysOppgave>`

Reply:
Ktor gjør noe her ja - ser intellij mener at _oppgaver_ er `Result<GosysOppgaver>,` men ved debug så er oppgaver `GosysOppgaver` på linje 51

Reply:
Hva jeg leser flere steder så anbefales det vel strengt tatt ikke å benytte _runCatching_, men heller normal try/catch for de absolutt fleste tilfeller - og spesifikt for forretningskode.

Reply:
Oppdatert. Også fjernet logging i `onFailure` her, da den dupliserer loggingen i klienten